src/main/python/README.md

```diff
@@ -2,7 +2,17 @@
 
 **starlake-airflow** is the **[Starlake](https://starlake-ai.github.io/starlake/index.html)** Python Distribution for **Airflow**.
 
-It is recommended to use it in combinaison with **[starlake dag generation](https://starlake-ai.github.io/starlake/docs/concepts/orchestration)**, but can be used directly as is in your **DAGs**.
+It is recommended to use it in combination with  **[starlake dag generation](https://starlake-ai.github.io/starlake/docs/concepts/orchestration)**, but can be used directly as is in your **DAGs**.
+
+## Prerequisites
+
+Before installing `starlake-airflow`, ensure the following minimum versions are installed on your system:
+
+- starlake 1.0.0
+- Python: 3.8 or higher
+- Apache Airflow: 2.4 or higher
```

Comment:
Actually to support BashSensor operator with retry_exit_code, Apache Airflow v 2.6 or higher should be required.
For older version, the option retry_on_failure for AirflowStarlakeCloudRunJob will fail

Reply:
I've put the versions for illustration. I've put 2.4 in reference to the introduction of data aware scheduling. 
```suggestion
- Apache Airflow: 2.6 or higher
```

Reply:
@fupelaqu is it ok for python version?